runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -205,6 +206,12 @@ public void onAnimationChanged(AnimationChanged event)
 			case USING_GILDED_ALTAR:
 			/* Farming */
 			case FARMING_MIX_ULTRACOMPOST:
+			/* Construction */
+			case HOME_MAKE_TABLET:
+				resetTimers();
```

Comment:
you dont need to re-do the lastAnimating and reset code here, just don't add break and let it fall-through